sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -645,6 +645,16 @@ case class DescribeTableCommand(
       } else if (isExtended) {
         describeFormattedTableInfo(metadata, result)
       }
+
+      // If any columns have default values, append them to the result.
+      if (metadata.schema.fields.exists(_.metadata.contains(
+        ResolveDefaultColumns.CURRENT_DEFAULT_COLUMN_METADATA_KEY))) {
+        append(result, "", "", "")
+        append(result, "# Column Default Information", "", "")
+        metadata.schema.foreach { column =>
+          column.getCurrentDefaultValue().map(append(result, column.name, "", _))
```

Comment:
Shall we fill the data type column to make it easier to read?
<img width="591" alt="image" src="https://user-images.githubusercontent.com/1097932/213785998-09d5d34e-8a7b-4ba5-b8c2-6cab52fdc02a.png">

Reply:
good suggestion, this is done!